.gitlab-ci.yml

```diff
@@ -16,6 +16,8 @@ variables:
   CARGO_INCREMENTAL:               0
   CI_SERVER_NAME:                  "GitLab CI"
   REGISTRY:                        registry.parity.io/parity/infrastructure/scripts
+  ALL_CRATES:                      "core alloc utils lang2 lang2/macro cli"
+  WASM_CRATES:                     "core alloc utils lang2 lang2/macro"
```

Comment:
Could you maybe even reuse `WASM_CRATES` in `ALL_CRATES` by using `$WASM_CRATES`?

Reply:
GitLab can't set lists to variables. This way it's explanatory enough.